### DISCUSSION TITLE
Any plans for IA-SSD?

### DISCUSSION THREAD

**Discussion by san9569** [open-mmlab/mmdetection3d#2623] [Ideas] [answered]
Hi all.

Thank you for greatly contributing to the 3D detection community!

Do you have any plans to add the [IA-SSD](https://arxiv.org/abs/2203.11139)?
It has a good balance of speed and accuracy and shows promising performance.

Best,
Sangjin

**Comment by sunjiahao1999** [comment, marked as answer]
We have no recent plans to support this model, but we welcome Pull Requests from those who are interested in it.

**Reply by san9569** [reply]
Thank you for replying!

I wonder if you have ever tried merging a project based on OpenPCdet with MMDetection3D.

I'm trying that but I'm not sure what I'm doing because I don't have any guidelines.

I really appreciate any help you can provide.
